perceval/backends/core/github.py

```diff
@@ -412,7 +412,14 @@ def __get_pull_review_comments(self, pr_number):
 
             for comment in json.loads(raw_comments):
                 comment_id = comment.get('id')
-                comment['user_data'] = self.__get_user(comment['user']['login'])
+
+                if 'user' not in comment or not comment['user']:
```

Comment:
I would change the code in the following way to simplify the if condition. Furthermore, it's probably better to provide the API link in the log message.
```
user = comment.get('user', None)
if not user:
      logger.warning("Missing user info for %s", comment['url'])
      comment['user_data'] = None
else:
      comment['user_data'] = self.__get_user(user['login'])
```

Reply:
Yes sure @valeriocos, this is more efficient code. I would change it to this. 
Also, will modify the log message to include the API link.